src/zope/app/form/browser/tests/test_form.py

```diff
@@ -21,19 +21,31 @@
 from zope.schema.interfaces import ITextLine
 from zope.component import testing
 
-from zope.app.testing import ztapi
+import zope.app.form.testing as ztapi
 
 from zope.app.form.browser import TextWidget
 from zope.formlib.interfaces import IInputWidget
 
+
+from zope.testing import renormalizing
+
+import re
+checker = renormalizing.RENormalizing([
+    (re.compile("u('.*?')"), r"\1"),
+    (re.compile('u(".*?")'), r"\1"),
+    # Python 3 adds module name to exceptions.
+    (re.compile('zope.configuration.xmlconfig.ZopeXMLConfigurationError'), 'ZopeXMLConfigurationError'),
```

Comment:
Interesting, I wasn't aware of that flag. 

Using it will require more changes to the doctests, though, to use fully-qualified dotted names in all cases. That might or might not be a good thing. Unless you object, I would just as soon leave things the way they are for right now.